src/tooltip/tooltip.js

```diff
@@ -339,6 +347,72 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
   }];
 })
 
+// This is mostly ngInclude code but with a custom scope
+.directive( 'tooltipTemplateTransclude', [
```

Comment:
Maybe this should be split off into its own component as a helper directive, similar to the dateParser service.